### DISCUSSION TITLE
Layouts with sub folders. #Nuxt3

### DISCUSSION THREAD

**Discussion by thingsneverchange** [nuxt/nuxt#18066] [Questions]
Hello, I am right from Nuxt2! 

A lot of things I am accustomed on Nuxt2 don't seem to work (I've just started to dig in though!) The main thing is the layout. In Nuxt2, it should work fine as beow:

```
<script>
export default{
  layout: 'folder/main',
  data() {
    return {
      message: 'Hello Vue.js!'
    }
  }
}
</script>
```

But this doesn't seem to work on Nuxt3, giving me an error:
**Invalid layout `folder/main` selected.**

Is there any workaround for this? I am sure a lot of enthusiasts here must have been using it on Nuxt2.

Thanks.

**Comment by lucpotage** [comment]
Hi! Did you try with `main` instead? I think the property corresponds to the layout name, not the layout file path.
